src/Bridge/ExceptionListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+declare(strict_types=1);
+
+namespace Jaeger\Symfony\Bridge;
+
+use Jaeger\Span\SpanManagerInterface;
+use Jaeger\Tag\ErrorTag;
+use Jaeger\Tracer\TracerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class ExceptionListener implements EventSubscriberInterface
+{
+    private $tracer;
+    private $spanManager;
+    private $globalSpanHandler;
+    private $exceptionExist;
+
+    public function __construct(
+        TracerInterface $tracer,
+        SpanManagerInterface $spanManager,
+        GlobalSpanHandler $globalSpanHandler
+    ) {
+        $this->tracer = $tracer;
+        $this->spanManager = $spanManager;
+        $this->globalSpanHandler = $globalSpanHandler;
+        $this->exceptionExist = false;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            ExceptionEvent::class => ['onKernelException', 0],
+            RequestEvent::class => ['onRequest', 28],
+            TerminateEvent::class => ['onTerminate', 4097],
```

Comment:
Split the priority in half across the next power of 2, e.g. 4096 + (8192 - 4096) / 2 = 6144. This way other people will have a chance to inject other Listeners before and after yours and prior to the existing ones

Reply:
I considered to remove this listener because it not needed.